.github/workflows/package-build.yml

```diff
@@ -25,6 +25,10 @@ on:
         description: The type of package to build (deb | rpm).
         type: string
         required: true
+      ref:
```

Comment:
Won't this workflow run in the same context as its parent (e.g. have the same implicit `ref`)?